File: Java/Sample004.java
import java.io.BufferedReader;
import java.io.InputStreamReader;

public class Sample004 {
    public static void main(String[] args) throws Exception {
        int seibetu;
        String namae = "";

        BufferedReader bufferedReader = new BufferedReader(new InputStreamReader(System.in));

        System.out.print("お名前は？：");
        namae = bufferedReader.readLine();

        System.out.println("性別は？：");
        System.out.println("1：男性");
        System.out.println("2：女性");
        seibetu = Integer.parseInt(bufferedReader.readLine());

        if( seibetu == 1 ) {
            System.out.println(namae + "くん、こんにちは！");
        } else if( seibetu == 2 ) {
            System.out.println(namae + "ちゃん、こんにちは！");
        }
    }
}

File: Python/sample004.py
namae = input("お名前は？：")

print("性別は？：")
print("1：男性")
print("2：女性")

seibetu = int(input())

if seibetu == 1:
  print(f"{namae}くん、こんにちは！")
elif seibetu == 2:
  print(f"{namae}ちゃん、こんにちは！")
